src/notation/inotation.h

```diff
@@ -23,6 +23,7 @@
 #define MU_NOTATION_INOTATION_H
 
 #include <QString>
+#include <QQuickItem>
```

Comment:
I don't think this should be in `inotation.h`...

Reply:
OK